go-controller/pkg/ovn/common.go

```diff
@@ -221,6 +225,21 @@ func (oc *Controller) getIPFromOvnAnnotation(ovnAnnotation string) string {
 	return ipAddressMask[0]
 }
 
+func (oc *Controller) getAnnotationMap(ovnAnnotation string) (map[string]string, error) {
```

Comment:
@alexanderConstantinescu instead of this function, we can now use:

`annotation, err := util.UnmarshalPodAnnotation(pod.Annotations["ovn"])`

Reply:
Thanks for the comment. I have modified the code accordingly. Just wanted to raise the attention to the fact that: `UnmarshalPodAnnotation` returns the IP address in CIDR format, where as `getIPFromOvnAnnotation` did not. I am not sure which is the right format to use when passing this off to the OVN ctl binaries, or if that even matters to them?